docs/gitops/README.md

```diff
@@ -19,6 +19,8 @@ oc apply -f docs/gitops/application.yaml
 
 # Grant admin permission to openshift gitops controller in knative-eventing
 oc adm policy add-role-to-user admin system:serviceaccount:openshift-gitops:openshift-gitops-argocd-application-controller -n knative-eventing
+oc adm policy add-role-to-user admin system:serviceaccount:openshift-gitops:openshift-gitops-argocd-application-controller -n knative-serving
```

Comment:
I'm not familiar with these docs. What is the purpose of those?

Reply:
By default gitops operator can only access its own namespaces, so you need to have that to enable management for other ns in a multi-tenancy setup, see [here](https://github.com/redhat-developer/gitops-operator/issues/116).

Reply:
Ok, but the document (the markdown) itself is used where? Or in what context?